### PR TITLE
XNET-109 | Initial PR

### PR DESCRIPTION
**Pull request by Pobepto**
# Description
https://soramitsu.atlassian.net/browse/XNET-109

### CODE REVIEW

**Review by antonkhvorov — APPROVED** [review]

File: src/App.vue [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+<script>
+import axios from 'axios'
+import BN from 'bignumber.js'
+import format from 'date-fns/format'
+
+export default {
+  name: 'app',
+  data () {
+    return {
+      URL: process.env.NODE_ENV === 'development'
```
Comment (Asmadek) [inline src/App.vue:66]: Maybe better to store URL as environment variable?

**Review by Asmadek — APPROVED** [review]
One comment

**Review by BulatSaif — APPROVED** [review]
CI works

File: src/App.vue (hunk 1 above)
Comment (Pobepto) [reply src/App.vue:66]: Next PR will be about it

**Pull request merged**